resources/deployments/mongo/deployments.go

```diff
@@ -261,9 +262,20 @@ func buildStatusQuery(status deployments.StatusQuery) bson.M {
 					bson.M{
 						buildStatusKey(deployments.DeviceDeploymentStatusRebooting): gt0,
 					},
+					bson.M{
+						"$and": []bson.M{
+							bson.M{
+								buildStatusKey(deployments.DeviceDeploymentStatusAlreadyInst): gt0,
```

Comment:
that needs to cover already installed, success, failure, noimage (all of them are finished with with pending > 0 are in progress)

dammit those cases are growing, btw that function is covered with unit test, maybe it's a good option to add cases there